Esri/arcgis-maps-sdk-swift-samples
# [New] Add ENC exchange set

## Description

This PR implements **`Add ENC Exchange Set`** in `Layers` category.

`swift/issues/5512`

## How To Test

Open view and wait for ENC data to download, once loaded it should load onto the map and render.

## Screenshots

![add-enc-exchange-set](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/13937686/501ab950-801a-48d2-aadb-76e4c3da2f3f)
